app/app.py

```diff
@@ -50,6 +53,9 @@ def request(self, flow: HTTPFlow):
             flow.request.port = 443
 
     def response(self, flow: HTTPFlow):
+        # Log requests
+        logger.info(f'GET {flow.request.url}: HTTP {"-" if flow.response is None else flow.response.status_code}')
```

Comment:
Do we know for sure that it is a `GET` request?

Reply:
Oh, good point, this should use the actual method.